server/src/main/java/io/spine/server/type/CommandClass.java

```diff
@@ -77,18 +76,22 @@ public static CommandClass of(Message commandOrMessage) {
         return new CommandClass(commandMessage.getClass(), typeUrl);
     }
 
-    /** Creates immutable set of {@code CommandClass} from the passed set. */
-    public static Set<CommandClass> setOf(Iterable<Class<? extends CommandMessage>> classes) {
+    /**
+     * Creates immutable set of {@code CommandClass} from the passed set.
```

Comment:
Please change the doc here as well.
```suggestion
     * Creates a set of {@code CommandClass} from the passed set.
```